test/e2e/container_iface_name_test.go

```diff
@@ -0,0 +1,600 @@
+package integration
+
+import (
+	"os"
+	"path/filepath"
+
+	. "github.com/containers/podman/v4/test/utils"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func createNetworkDevice(name string) {
+	session := SystemExec("ip", []string{"link", "add", name, "type", "bridge"})
+	session.WaitWithDefaultTimeout()
```

Comment:
this is not checking the exit code so it can fail silently, it needs
`Expect(session).To(Exit(0))`